fea-rs/src/compile/validate.rs

```diff
@@ -613,10 +613,10 @@ impl<'a, V: VariationInfo> ValidationCtx<'a, V> {
                         node.keyword().range(),
                         "cvParameters block only valid in cv01-cv99 features",
                     );
-                } else if !is_first_item {
+                } else if has_seen_rule {
                     self.error(
                         node.keyword().range(),
-                        "cvParameters must be first statement in feature block",
+                        "cvParameters must precede any rules",
```

Comment:
Rules are not required, and the statement "must precede rules" implies that rules must always follow. The phrasing "must precede any rules" is more accurate, because it indicates that the statement may also precede no rules at all.

Reply:
You only emit this when it *doesn't* precede rules so the no rules at all case seems not to apply? 

Anotehr suggestion would be to state what the *problem* is, e.g. `cvParameters cannot come after rules`